third-party/dms-guild/class-cook.xml

```diff
@@ -0,0 +1,1379 @@
+      <ul>
+        <li>Monstrosity. You have advantage on Strength, Dexterity,and Constitution saving throws against magic.</li>
+        <li>Ooze. You gain blindsight out to a range of 10 feet.</li>
+        <li>Plant. While you are standing in direct sunlight, you have advantage on Strength and Constitution checks, and whenever you would regain hit points, treat any dice rolled as having rolled their maximum value.</li>
+        <li>Aberration. You gain darkvision out to a range of 60 feet and telepathy out to a range of 60 feet.</li>
+        <li>Dragon. You gain a flying speed of 60 feet. If you end your turn in the air, and nothing is keeping you aloft, you fall.</li>
+        <li>Fiend. You have advantage on all Charisma and Intelligence checks and saving throws.</li>
+        <li>Giant. You gain the benefits of the Enlarge spell.</li>
+      </ul>
+    </description>
+    <sheet>
+      <description>When you or an ally kills an enemy of CR 1 or higher you can make a feast out of it's body during the next rest for {{feast:number}} creatures.</description>
+    </sheet>
+    <rules>
+      <stat name="feast:number" value="5" />
+      <stat name="feast:number" value="Constitution Modifier" />
+    </rules>
+  </element>
+  <element name="Autonomous Bag" type="Archetype Feature" source="The Cook Class" id="ID_DMSG_COOK_ARCHETYPE_FEATURE_MONSTER_MASHER_AUTONOMOUS_BAG">
+    <description>
+      <p>At 6th level, you modify your Cook's Bag to use the vitality coursing through a recently taken piece of flesh to cook it into a morsel.</p>
+      <p>When you deal slashing or piercing damage to a hostile creature that isn't a humanoid, celestial, undead, construct, or fey, with a melee weapon attack, you can cut off a chunk of meat and place it into your Cook's Bag as part of the same attack. The chunk is immediately seasoned and cooked into the following morsel by your Cook's Bag. You can't have more than one morsel of this type in your bag at a time.</p>
+      <p>Your Cook's bag can season and cook the following morsel a number of times equal to your Constitution modifier (minimum of once). You regain expended uses when you finish a long rest.</p>
+    </description>
+    <sheet>
+      <description>When you deal piercing or slashing damage you can cook 1 Fresh Cut Morsel as a bonus action. You can hold only 1 Fresh Cut and can cook Fresh Cut {{constitution:modifier}} times per long rest.</description>
+    </sheet>
+    <rules>
+      <grant type="Class Feature" name="ID_DMSG_COOK_CLASS_FEATURE_COOK_MORSEL_FRESH_CUT" />
```

Comment:
This grant throws an error for me as it can't find this.

Reply:
Fixed it thanks for catching it!